tests/core/pyspec/eth2spec/test/helpers/light_client.py

```diff
@@ -42,13 +42,18 @@ def get_sync_aggregate(spec, state, signature_slot=None):
     committee_indices = compute_committee_indices(spec, signature_state)
```

Comment:
note to myself: out of the scope of this PR, `compute_committee_indices` helper doesn't need `spec` parameter at all.